awx/ui_next/src/screens/Template/Survey/SurveyPreviewModal.jsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react';
+import { withI18n } from '@lingui/react';
+import styled from 'styled-components';
+import { t } from '@lingui/macro';
+
+import {
+  Chip as _Chip,
+  Form,
+  FormGroup,
+  Modal,
+  TextInput,
+  TextArea,
+  InputGroup,
+  InputGroupText as _InputGroupText,
+  Button,
+  Select,
+} from '@patternfly/react-core';
+
+const InputGroupText = styled(_InputGroupText)`
+  width: 100%;
+`;
+const Chip = styled(_Chip)`
+  margin-right: 5px;
+`;
+function SurveyPreviewModal({
+  questions,
+  isPreviewModalOpen,
+  onToggleModalOpen,
+  i18n,
+}) {
+  const [isPassword, setIsPassword] = useState(true);
+  return (
+    <Modal
+      title={i18n._(t`Survey Preview`)}
+      isOpen={isPreviewModalOpen}
+      onClose={() => onToggleModalOpen(false)}
+      isSmall
+    >
+      <Form>
+        {questions.map(q => (
+          <div key={q.variable}>
+            <strong>{q.question_name}</strong>
+            {['text', 'integer', 'float'].includes(q.type) && (
+              <FormGroup fieldId="text">
```

Comment:
So I think these fieldId's need to actually match an ID on the field.  For example:

```
              <FormGroup fieldId="survey-preview-textarea">
                <TextArea
                  id="survey-preview-textarea"
                  value={q.default}
                  aria-label={i18n._(t`Text Area`)}
                  disabled
                />
              </FormGroup>
```